### PR TITLE
Adding Donor Role whilst creating ANY user /added seedRole migration to db

### PR DESCRIPTION
**Pull request by badekk**
## Adding Donor Role whilst creating ANY user /added seedRole migration to db

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

- [ ] Unit test(s)
- [X] Integration test(s)
- [X] Tested manually

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings
- [X] I have added frontend code and checked it for mobile & desktop

### CODE REVIEW

**Comment by mar-pan** [comment]
Task linked: [CU-862j6m7aq Add role in postlogin](https://app.clickup.com/t/862j6m7aq)

File: src/backend/AuthService/Actions/AuthActions/PostLogin/UserCreationService.cs [hunk 1]
```diff
@@ -83,5 +100,19 @@ private static Auth0User CreateAuth0User(CurrentUserInfo currentAuth0UserInfo, U
...
+        {
+            return new UserRole
+            {
+                RoleId = 2,
+                User = user,
+                UserId = user.Id,
+                UpdateUser = serviceUser,
+                UpdateUserId = serviceUser.Id,
+                UpdatedAt = DateTime.UtcNow,
+                Status = Core.Database.Enums.DbEntityStatus.Active
```
Comment (h4570) [inline src/backend/AuthService/Actions/AuthActions/PostLogin/UserCreationService.cs:114]: `Core.Database.Enums.DbEntityStatus.Active` -> `DbEntityStatus.Active` i dodaj usinga

**Review by h4570 — CHANGES_REQUESTED** [review]
1 koment + SonarQube + popraw "YOUR PR NAME - CHANGE ME"

**Review by h4570 — APPROVED** [review]

**Comment by h4570** [comment]
Możesz mergować

**Pull request merged**